modules/viewing-the-image-pull-source.adoc

```diff
@@ -0,0 +1,52 @@
+// Module included in the following assemblies:
+//
+// *installing/validating-an-installation.adoc
+
+[id="viewing-the-image-pull-source_{context}"]
+= Viewing the image pull source
+
+You can view the source of your pulled images by using the CRI-O logs on your nodes. 
+
+.Prerequisites
+
+* You have access to the cluster as a user with the `cluster-admin` role.
+
+.Procedure
+
+* Review the CRI-O logs for a master node or for a worker node:
++
+[source,terminal]
+----
+$  oc adm node-logs ci-ln-jb5m1vb-f76d1-cfzck-master-0 -u crio
+----
++
+.Example output
++
+The `Trying to access` log entry indicates where the images are being pulled from.
++
+[source,terminal]
+----
+...
+Mar 17 02:52:50 ip-10-0-138-140.ec2.internal crio[1366]: time="2021-08-05 10:33:21.594930907Z" level=info msg="Pulling image: quay.io/openshift-release-dev/ocp-release:4.8.4-ppc64le" id=abcd713b-d0e1-4844-ac1c-474c5b60c07c name=/runtime.v1alpha2.ImageService/PullImage
+Mar 17 02:52:50 ip-10-0-138-140.ec2.internal crio[1484]: time="2021-03-17 02:52:50.194341109Z" level=info msg="Trying to access \"li0317gcp1.mirror-registry.qe.gcp.devcluster.openshift.com:5000/ocp/release@sha256:1926eae7cacb9c00f142ec98b00628970e974284b6ddaf9a6a086cb9af7a6c31\""
+Mar 17 02:52:50 ip-10-0-138-140.ec2.internal crio[1484]: time="2021-03-17 02:52:50.226788351Z" level=info msg="Trying to access \"li0317gcp1.mirror-registry.qe.gcp.devcluster.openshift.com:5000/ocp/release@sha256:1926eae7cacb9c00f142ec98b00628970e974284b6ddaf9a6a086cb9af7a6c31\""
+...
+----
++
+[NOTE]
+====
+The log might show the image pull list twice, as shown in the preceding example. 
+====
++
+If your `ImageContentSourcePolicy` object lists multiple mirrors, {product-title} attempts to pull the images in the order listed in the configuration, for example:
```

Comment:
Can you tell me  what populates {product-title} ?

Reply:
@lyman9966 {product-title} is either OpenShift Conatiner Platform or OKD.